packages/cozy-intent/src/api/services/NativeService.ts

```diff
@@ -81,11 +81,9 @@ export class NativeService {
   public tryEmit = async (event: NativeEvent): Promise<void> => {
     const parsedEvent = this.parseNativeEvent(event)
 
-    if (!this.isPostMeMessage(parsedEvent)) return
+    if (this.isInitMessage(parsedEvent)) await this.tryInit(event)
 
-    return this.isInitMessage(parsedEvent)
-      ? await this.tryInit(event)
-      : this.tryOnMessage(event, parsedEvent)
+    if (this.isPostMeMessage(parsedEvent)) this.tryOnMessage(event, parsedEvent)
```

Comment:
let's test this feature to document that feature. I don't understand why this would fix the CI